### ISSUE TITLE
Do webhooks work for srophe/srophe-eXist-app repo?

### ISSUE THREAD

**Issue by davidamichelson** [srophe/srophe-eXist-app#710]
I just pushed some changes to srophe-eXist-app/srophe-app/bibl but they are not showing up on the dev server.

**Comment by wsalesky** [comment]
No, they don't work for the app... I can set them up. In the meantime I will push your changes.

**Comment by wsalesky** [comment]
Okay. They are enabled. Please make sure your records are well formed or they will not load. Let me know if there are any issues.
